src/SignalR/common/Http.Connections/ref/Microsoft.AspNetCore.Http.Connections.netcoreapp3.0.cs

```diff
@@ -100,8 +100,11 @@ public partial class HttpConnectionContext : Microsoft.AspNetCore.Connections.Co
         public System.Threading.SemaphoreSlim WriteLock { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
         [System.Diagnostics.DebuggerStepThroughAttribute]
         public System.Threading.Tasks.Task DisposeAsync(bool closeGracefully = false) { throw null; }
+        public void MarkInactive() { }
```

Comment:
@davidfowl What's the deal with reference assemblies for pubternal classes? Do we really have to keep back compat? If so, should we just make everything new internal?